admin_manual/configuration/files/external_storage/google.rst

```diff
@@ -14,97 +14,135 @@ If you already have a Google account, such as Groups, Drive, or Mail, you can
-then enter your app name. **Authorized JavaScript Origins** is your root domain, 
-for example ``https://example.com``, without a trailing slash. You need two 
-**Authorized Redirect URIs**, and they must be in this form::
+.. figure:: images/google_drive/010
+   :alt: Access type and Data
 
+**Authorized JavaScript Origins** is your root domain, 
+for example ``https://example.com``, without a trailing slash. 
+Examples::
+
+  https://example.com
+  http://example.com
+  IP/owncloud  
+
+You need to configure **Authorized Redirect URIs**, and they must be in this form::
+
+  https://example.com/owncloud/index.php/settings/admin?sectionid=storage
   https://example.com/owncloud/index.php/settings/personal?sectionid=storage
-  https://example.com/owncloud/index.php/settings/admin?sectionid=storage 
-  
-Replace ``https://example.com/owncloud/`` with your own ownCloud server URL, 
-then click **Create**.
 
-.. figure:: images/google-drive9.png
-   :alt: Creating client credentials.
+If you configuring storage as an Administrator - choose the admin URI, if you are a user and configuring a storage
+- pick the personal URI.
 
-Now Google reveals to you your **Client ID** and **Client Secret**. Click 
-**OK**.
+If you are not sure what your exact URIs are - here is a quick guide to figure it out.
 
-.. figure:: images/google-drive10.png
-   :alt: Client ID and client secret.
+**Authorized JavaScript Origins**
 
-You can see these anytime in your Google console; just click on your app name 
-to see complete information.   
+This is just the address you access you ownCloud on, where you see the login screen.
 
-.. figure:: images/google-drive11.png
-   :alt: Client ID and client secret
-   
-Now you have everything you need to mount your Google Drive in ownCloud. 
+.. figure:: images/google_drive/011
+   :alt: text
+
+**Authorized Redirect URIs**
+
+If you have not already enabled the google drive storage, here is how you do it:
+
+.. figure:: images/google_drive/011
+   :alt: Login in ownCloud
+
+.. figure:: images/google_drive/012
+   :alt: Go to Storage in the Settings
 
-Go to the External Storage section of your Admin page, create your new folder 
-name, enter the Client ID and Client Secret. If you wish limit access to a single folder, simply enter the path to the desired folder, separated by '/'. Finally, click **Grant Access**. Your 
+.. figure:: images/google_drive/013
+   :alt: Enable external Storage
+
+.. figure:: images/google_drive/014
+   :alt: Select Google Drive from dropdown menu
+
+.. figure:: images/google_drive/015
+   :alt: Now you have your Google Drive App enabled
+
+.. figure:: images/google_drive/016
+   :alt: The URL from this page is the one you have to enter in the **Authorized Redirect URIs**
+
+Here is the correct result:
+
+.. figure:: images/google_drive/017
+   :alt: Client ID
+
+Now we have to create a consent screen. This is the information in the screen 
```

Comment:
Please join the second sentence on this line with the parts of it on lines 114, & 115.